### PR TITLE
DSE 6.8.29 release notes

### PR DESCRIPTION
**Pull request by tiagomlalves**

### CODE REVIEW

File: DSE_6.8_Release_Notes.md [hunk 1]
```diff
@@ -55,7 +79,6 @@ If you're developing applications, please refer to the [Java Driver documentatio
 * Fixed an issue where EMPTY_LAST_CONTENT would no be written to the stream when the connection was closed. (DSP-22671)
 
 ## 6.8.27 DSE Security
-* Fixed an issue that caused authentication to fail in a cluster of analytics nodes in case of a version upgrade, where the new version of node could not authenticate requests from an old version of node. (DSP-22723)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:58]: Now I wonder how did it even land here?

File: DSE_6.8_Release_Notes.md [hunk 2]
```diff
@@ -47,6 +70,7 @@ If you're developing applications, please refer to the [Java Driver documentatio
 
 ## 6.8.27 DSE Graph
 * Fixed GraphOLAP spark connection problem in the multi-network cloud environments when GPFS is not used. (DSP-22707)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:72]: I know it's a previous release but it seems sensible to include the full name GossipingPropertyFileSnitch as well as the abbreviation, for easier searchability.

**Review by szymon-miezal — APPROVED** [review]

File: DSE_6.8_Release_Notes.md (hunk 2 above)
Comment (tlasica) [reply DSE_6.8_Release_Notes.md:72]: Indeed. But people usually use `GPFS` abbreviation.
See history of the original ticket (DSP-22707) for "why sometimes abbreviations are better and we should check emails".

Comment (szymon-miezal) [reply DSE_6.8_Release_Notes.md:72]: after finding `General Parallel File System` I give up and vote to stick to the abbreviation.

Comment (tiagomlalves) [reply DSE_6.8_Release_Notes.md:72]: Why not write:

> Fixed GraphOLAP spark connection problem in the multi-network cloud environments when GossipingPropertyFileSnitch (GPFS) is not used. (DSP-22707)

?

**Pull request merged**